plugins/generator-1.18.1/forge-1.18.1/procedures/compare_mcblocks_exact.java.ftl

```diff
@@ -0,0 +1,2 @@
+<#include "mcitems.ftl">

```

Comment:
Exact procedure blocks are no longer present in this branch, you can and should remove them

Reply:
Oh, the file was present in Forge 1.17.1 when I did this PR.

Reply:
You are right, but now they can be removed as I made a converter to remove these. They were only there for 1.12.2 reasons

Reply:
I just updated the branch and I see they have been removed.